ee/maintained-apps/inputs/homebrew/scripts/camtasia_uninstall.sh

```
@@ -0,0 +1,41 @@
+  local file_name="$(basename "${target_file}")"
+
+  if [[ -e "$target_file" ]]; then
+    echo "removing $target_file."
+    mv -f "$target_file" "$trash/${file_name}_${timestamp}_${rand}"
+  else
+    echo "$target_file doesn't exist."
+  fi
+}
+
+sudo rm -rf "$APPDIR/Camtasia.app"
+trash $LOGGED_IN_USER '/Users/Shared/TechSmith/Camtasia'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)